wotw_seedgen/areas.wotw

```diff
@@ -11295,26 +11301,29 @@ anchor EastPools.LupoArea at -1409, -4163:  # Checkpoint to the left of Lupo
     unsafe, EastPools.LupoPurpleFloor: Damage=180
   conn EastPools.CentralRoom:
     kii, EastPools.LupoPurpleFloor, EastPools.CentralRoomPurpleWall: WaterDash, Damage=100
-    unsafe, EastPools.LupoPurpleFloor, EastPools.CentralRoomPurpleWall: Damage=240
+    unsafe, EastPools.LupoPurpleFloor, EastPools.CentralRoomPurpleWall:
+      Damage=240
+      WaterDash, Damage=80
 
 anchor EastPools.WaterdashArena at -1348, -4139:  # Checkpoint at the right of the pools arena
   refill Checkpoint
-  
+
   # There are two Arenas here: you get one if you enter without WaterDash and another if you enter with it.
   # This path is assuming the worse case of getting the strictly harder WaterDash arena, but doesn't list WaterDash explicitly because you don't need it unless you have it.
   # And any other way would lock basically all of Pools behind WaterDash. So it makes sense, right?
   state EastPools.FightArenas:
     moki: Regenerate, Damage=20, Water, WaterBreath, Combat=4xCrab+Tentacle+2xSkeeto, Bash
     gorlek: Regenerate, Damage=20, Water, Combat=4xCrab+Tentacle+2xSkeeto, Bash
     unsafe: Water, Combat=4xCrab+Tentacle+2xSkeeto, Launch OR Dash OR Damage=20  # Land on the left then jump under the purple wall to correct the torpedo angle. launch in order to avoid the blast damage
-  
+
   pickup EastPools.FightRoomHC:  # The HC only gets unlocked if you finish the alternate WaterDash arena; FightArenas doesn't have WaterDash as requirement included for reasons explained above, so it's needed here
     moki, EastPools.FightArenas, WaterDash:
       DoubleJump, Dash OR Grapple
       Bash, Grenade=1
       Launch
-    gorlek, EastPools.FightArenas, WaterDash: Water OR Damage=20  # WaterDash on the bubble
-    kii, EastPools.FightArenas, WaterDash: DoubleJump  #@validator that might be gorlek? gorlek for sure with tjump
+    gorlek, EastPools.FightArenas, WaterDash:
+      Water OR Damage=20  # WaterDash on the bubble
+      DoubleJump
```

Comment:
I think gorlek would be better with TripleJump. It only seems to work leniently when jumping from the left side and using higher ground as starting point isn't intuitive to all players.

Reply:
I added triple jump in Gorlek, and moved double jump back to Kii.